### PR TITLE
docs: move tutorial to another section

### PR DESCRIPTION
**Pull request by zeljkobekcic**
This branch moves the connector tutorial to another section. Meaning that on https://docs.datalbry.io/ would be another section called `Tutorials` which contains the said tutorial.

Additionally the links in the README.md have been fixed (not existing markdown files). Now they point instead to the templates in `templates`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,9 @@ The `Connector-SDK` is a sophisticated framework to develop `Connector Integrati
 A `Connector Integration` is a application, fetching data from a system, transform it briefly and put it into 
 another system. It can be considered as an `ETL` / `ELT` tool, while not enabling the user to transform the data.  
 
-* [Overview](/docs/QUICKSTART.mdx)
-* [Getting Started](/docs/QUICKSTART.mdx)
-* [Concept](/docs/CONCEPT.mdx)
-* [Testing](/docs/TESTING.mdx)
-* [Upgrade-Guide](/docs/UPGRADE_GUIDE.mdx)
+* [Overview](https://docs.datalbry.io/Integrations/Connector-SDK/OVERVIEW)
```
Comment (Lacritz) [inline README.md:8]: unnecessary, as they are relative to one another. We do not hardcode the path here.

File: README.md [hunk 2]
```diff
@@ -5,11 +5,9 @@ The `Connector-SDK` is a sophisticated framework to develop `Connector Integrati
...
+* [Getting Started](https://docs.datalbry.io/Integrations/Connector-SDK/QUICKSTART)
```
Comment (Lacritz) [inline README.md:9]: s.a.

File: README.md [hunk 3]
```diff
@@ -5,11 +5,9 @@ The `Connector-SDK` is a sophisticated framework to develop `Connector Integrati
...
+* [Concept](https://docs.datalbry.io/Integrations/Connector-SDK/CONCEPT)
```
Comment (Lacritz) [inline README.md:10]: s.a.

File: README.md (hunk 1 above)
Comment (zeljkobekcic) [reply README.md:8]: The `docs` directory does not exist anymore. Does it make sense to change it to `/templates/docs/` instead?

**Review by Lacritz — APPROVED** [review]

**Pull request merged**